server/src/main/java/org/elasticsearch/index/IndexMode.java

```diff
@@ -89,14 +95,12 @@ void validateWithOtherSettings(Map<Setting<?>, Object> settings) {
                     throw new IllegalArgumentException(error(unsupported));
                 }
             }
-            settingRequiresTimeSeries(settings, IndexMetadata.INDEX_ROUTING_PATH);
-            settingRequiresTimeSeries(settings, IndexSettings.TIME_SERIES_START_TIME);
-            settingRequiresTimeSeries(settings, IndexSettings.TIME_SERIES_END_TIME);
```

Comment:
Are `IndexSettings.TIME_SERIES_START_TIME` and `IndexSettings.TIME_SERIES_END_TIME` not required in the time_series index? I added the checking code to make these settings required. This is a requirement for the time_series index before. Now remove the check will change the requirement.  I think it's ok to make these settings not required, but we should check that if it satisfies the design of time_series index.

Reply:
They are no longer required, but if they are not there we just use the default values.

Reply:
It looks like if you leave it blank he's got it configured to cover -9999 to 9999. I wasn't super keen on having defaults for these a few months ago, but I suppose it's fine.

Reply:
My preference would be to support an entire range from Long.MIN_VALUE to Long.MAX_VALUE but we are not capable of parsing such dates. This is the entire range we can parse.